java/api/pom_template.xml

```diff
@@ -6,7 +6,7 @@
   <parent>
     <groupId>io.ray</groupId>
     <artifactId>ray-superpom</artifactId>
-    <version>2.0.0-SNAPSHOT</version>
+    <version>2.8.0</version>
```

Comment:
not sure if these ones need to be changed or not.. why they were 2.0.0-SNAPSHOT on master?

these seem like java version, rather than ray version strings.

Reply:
go/release-ray still has it in the instructions, and 2.7.1 commit has them too, so it's probably okay to keep